executor/common_linux.h

```diff
@@ -3284,7 +3284,10 @@ static void sandbox_common()
 	// CLONE_NEWNS/NEWCGROUP cause EINVAL on some systems,
 	// so we do them separately of clone in do_sandbox_namespace.
 	if (unshare(CLONE_NEWNS)) {
-		debug("unshare(CLONE_NEWNS): %d\n", errno);
+		fprintf(stderr, "unshare(CLONE_NEWNS): %d\n", errno);
```

Comment:
Please use error handling consistent with the all the error handling around (debug function for ignored errors). These 2 functions are not really any special to introduce a completely new error handling strategy.